apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -341,9 +382,16 @@ export class LemonadePipeline implements BasePipeline {
                     : t.user_name,
                 lemonadeEventId: eventConfig.externalId,
                 lemonadeTicketTypeId: t.type_id,
+                genericIssuanceEventId: eventConfig.genericIssuanceEventId,
+                // The 'as string' cast here is safe because we know that the
+                // ticket type exists, having earlier filtered out tickets for
+                // which the ticket type does not exist.
```

Comment:
There probably is, but it might require a bigger refactor. We have a lot of these "the pipeline has some configuration, and every time we access it we have to check that it really exists" things, and perhaps there's a better pattern for dealing with them.